src/videouplinkbandwidthpolicy/DefaultSimulcastUplinkPolicy.ts

```diff
@@ -82,160 +62,84 @@ export default class DefaultSimulcastUplinkPolicy implements SimulcastUplinkPoli
-        // still take a while to recover (as it needs to send padding until it reaches around
-        // 1000 kbps).
-        this.newActiveStreams = ActiveStreams.kHi;
-        newBitrates[0].maxBitrateKbps = 0;
-        newBitrates[1].maxBitrateKbps = this.hiTargetBitrateKbps;
-        newBitrates[2].maxBitrateKbps = 0;
-      } else if (this.numSenders <= 4 && this.lastUplinkBandwidthKbps >= this.hiDisabledRate) {
-        // E.g., 320x192+ (640x384)  + 1280x768
-        this.newActiveStreams = ActiveStreams.kHiAndLow;
-        newBitrates[0].maxBitrateKbps = this.lowTargetBitrateKbps;
-        newBitrates[1].maxBitrateKbps = 0;
-        newBitrates[2].maxBitrateKbps = this.hiTargetBitrateKbps;
-      } else if (this.lastUplinkBandwidthKbps >= this.midDisabledRate) {
-        // E.g., 320x192 + 640x384 + (1280x768)
-        this.newActiveStreams = ActiveStreams.kMidAndLow;
-        newBitrates[0].maxBitrateKbps =
-          this.lastUplinkBandwidthKbps >= this.midTargetBitrateKbps * 0.7
-            ? (this.lowTargetBitrateKbps * 2) / 3
-            : this.lowTargetBitrateKbps / 2;
-        newBitrates[1].maxBitrateKbps =
-          this.numSenders <= 6 ? this.midTargetBitrateKbps : this.midTargetBitrateKbps * 0.6;
-        newBitrates[2].maxBitrateKbps = 0;
-      } else {
-        // E.g., 320x192 + 640x384 + (1280x768)
-        this.newActiveStreams = ActiveStreams.kLow;
-        newBitrates[0].maxBitrateKbps = this.lowTargetBitrateKbps;
-        newBitrates[1].maxBitrateKbps = 0;
-        newBitrates[2].maxBitrateKbps = 0;
-      }
-      const bitrates: number[] = newBitrates.map((v, _i, _a) => {
-        return v.maxBitrateKbps;
-      });
-
-      this.newQualityMap = this.fillEncodingParamWithBitrates(bitrates);
-      if (!this.encodingParametersEqual()) {
-        this.logger.info(
-          `simulcast: policy:calculateEncodingParameters bw:${
-            this.lastUplinkBandwidthKbps
-          } numSources:${this.numSenders} shouldDisableSimulcast:${
-            this.shouldDisableSimulcast
-          } newQualityMap: ${this.getQualityMapString(this.newQualityMap)}`
-        );
-      }
+    this.newQualityMap = this.fillEncodingParamWithBitrates(bitrates);
+    if (!this.encodingParametersEqual()) {
+      this.logger.info(
+        `simulcast: policy:calculateEncodingParameters numSources:${
+          this.numSenders
+        } shouldDisableSimulcast:${
+          this.shouldDisableSimulcast
+        } newQualityMap: ${this.getQualityMapString(this.newQualityMap)}`
+      );
     }
     return this.newQualityMap;
   }
 
   chooseMediaTrackConstraints(): MediaTrackConstraints {
-    // Changing MediaTrackConstraints causes a restart of video input and possible small
-    // scaling changes.  Always use 720p for now
-    const trackConstraint: MediaTrackConstraints = {
-      width: { ideal: this.enableFhdVideo ? 1920 : 1280 },
-      height: { ideal: this.enableFhdVideo ? 1080 : 768 },
```

Comment:
This 768 value was the original issue. Leads to cropping. But 720 works just fine.